virtual_rainforest/models/animals/__init__.py

```diff
@@ -0,0 +1,10 @@
+# noqa: D205, D415
```

Comment:
The `__init__.py` file provides the docstring for `virtual_rainforest.models.animal` and so does need a docstring. That can be just a simple overview of the contents of the submodules. In the long run it probably could also give some linking theory about the model approach to help understand how it works, but not needed now.